dashboard/db/migrate/20160219091950_create_level_groups.rb

```diff
@@ -0,0 +1,20 @@
+class CreateLevelGroups < ActiveRecord::Migration
```

Comment:
did you actually mean to create a new table? It seems to be the same schema as Level (so it looks like LevelGroups should be stored as a Level with type = LevelGroup similar to the other subtypes of levels)